requirements-build.txt

```diff
@@ -10,14 +10,23 @@ cython==0.29.33
     # via -r requirements-build.in
 docutils==0.19
     # via -r requirements-build.in
+editables==0.3
+    # via hatchling
 flit-core==3.8.0
     # via -r requirements-build.in
+hatchling==1.13.0
+    # via -r requirements-build.in
```

Comment:
oh boy. hatchling depends on itself for building.

looking forward to see how this bad boy will behave downstream (hopefully no changes required)

Reply:
(not that this is any blocker. just saying)

Reply:
OMG docs don't have info on how to build it lol
![image](https://user-images.githubusercontent.com/6073215/227997265-76b0b70f-ba6b-45c5-93a3-d76f27da70a1.png)

Reply:
I guess we'll find out downstream soon enough! 🚀 💥